test/TestsHelper.js

```diff
@@ -117,8 +118,8 @@ TestsHelper.assertions = {
         return done(err);
       }
 
-      if (!expected || isEmpty(expected.flex)) {
-        const actualDoesNotContainData = isEmpty(actual) || isEmpty(actual.flex);
+      if (!expected) {
+        const actualDoesNotContainData = isEmpty(actual);
```

Comment:
Why the change?

Reply:
It seems that at some point this had broken some of the tests, but now they are passing, so I am reverting the change.

Reply:
Update: The change is needed, as I am passing as expected ``` {
  "flexListProfile": {
    "flex": {
      "domain": "app"
      "domainEntityId": "885f5d307afd4168bebca1a64f815c1e"
      "schemaVersion": 2
      "serviceId": "12378kdl2"
      "serviceName": "TestKinveyDatalink"
    }
  }
}``` and with the old version this will always execute ```expect(actualDoesNotContainData, `Setup at ${path} is empty.`).to.be.true;``` and the tests will always pass.